index.html

```diff
@@ -1,19 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+              <header class="header">
+                    <img class="karma_logo" src="img/karma-logo.svg" alt="karma logo"> 
+                        <!--navabar-->
+                        <nav> 
+                            <ul class="nav_list">
+                                <li class="nav_item">
+                                    <a class="nav_link" id= "meet_karma"href=""><strong>Meet Karma</strong></a>
+                                </li>
+                                <li class="nav_item">
+                                    <a class="nav_link" href="">How It Works</a>
+                                </li>
+                                <li>
+                                    <a class="nav_link" href="">Store</a>
+                                </li class="nav_item">
+                                <li>
+                                    <a class="nav_link" href="">Blog</a>
+                                </li class="nav_item">
+                                <li>
+                                    <a class="nav_link" href="">Help</a>
+                                </li class="nav_item"> 
+                                <li>
+                                    <a class="nav_link" href="">Login</a>
+                                </li>
+                            </ul>
+                        </nav>
+            </header>
+            <!--main-->
+             <main class="main">
+                 <div class="intro">
+                    <h1>Introducing Karma</h1>
+                    <h2>Bring WiFi with you, everywhere you go.</h2>
+                    <button>Learn More</button>
+                 </div>
+                 <!--Everyone Needs a little Karma-->
+                 <div class="everyone">
```

Comment:
Fight the urge to use divs for everything. There is always a better alternative - review the [semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp).